kp_registry/registry.py

```diff
@@ -28,6 +28,7 @@ def __init__(
 
     async def __aenter__(self):
         """Enter context."""
+        print(self.uri)
```

Comment:
Is this necessary, or is it a leftover from debugging?

Reply:
Oops, leftover